spec.bs

```diff
@@ -76,7 +76,7 @@ advertisement relevant to this interest to this user in the future. The user age
 <xmp class="idl">
 [SecureContext]
 partial interface Navigator {
-  Promise<undefined> joinAdInterestGroup(AuctionAdInterestGroup group, double durationSeconds);

```

Comment:
```
joinAdInterestGroup: ƒ joinAdInterestGroup()
length: 2
name: "joinAdInterestGroup"
```
This breaking change will land with 115, yes?

The migration strategy is to start including the new `lifetimeMs` attribute and check `joinAdInterestGroup()` arity to determine legacy invocation.

If a note is not to be included in the associated FLEDGE explainer `joinAdInterestGroup` update, a release notes table of breaking changes would be helpful.

Reply:
@dmdabbs This won't be a breaking change -- the plan is that we'll support these scenarios for M115:

1. Setting the old expiration via joinAdInterestGroup()
2. Setting the new lifetimeMs on the interest group
3. Setting both (for compatibility with older browsers -- lifetimeMs will take effect and the old expiration on joinAdInterestGroup() will be ignored).

I plan to update the explainer after landing the code changes. 

Eventually, we'll only support 2 and 3 above, but only once API users have migrated.

NOTE: The new version uses milliseconds instead of seconds.